ui/component/walletTipAmountSelector/view.jsx

```diff
@@ -143,7 +144,12 @@ function WalletTipAmountSelector(props: Props) {
       }
       //  if tip fiat tab
     } else {
-      if (amount < 1) {
+      regexp = RegExp(/^(\d*([.]\d{0,2})?)$/);
+      const validTipInput = regexp.test(String(amount));
+
+      if (!validTipInput) {
+        tipError = __('Amount must have no more than 2 decimal places');
+      } else if (amount < 1) {
         tipError = __('Amount must be at least one dollar');
       } else if (amount > 1000) {
```

Comment:
We can move magic numbers like 1000 to constants at the top such as MAX_FIAT_TIP